.github/workflows/plugins.yml

```diff
@@ -778,6 +778,23 @@ jobs:
         uses: ./.github/actions/testagent/logs
       - uses: codecov/codecov-action@v2
 
+  openai:
```

Comment:
The tests weren't running in CI at all! I never noticed since I only pushed code if tests passed locally.